src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1316,8 +1316,12 @@ def _validate_trusted_launch(namespace):
         return
 
     if not namespace.enable_vtpm or not namespace.enable_secure_boot:
```

Comment:
Since `namespace.enable_vtpm` already has the default value, so please do not add warning log for `enable_vtpm` any more